### PR TITLE
Added "Using Angular-PatternFly In Your Application" section 

### PR DESCRIPTION
**Pull request by dtaylor113**
...to README.md

### CODE REVIEW

**Comment by dtaylor113** [comment]
Hi, do we require angular-bootstrap now?

<script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.min.js"></script>

File: README.md [hunk 1]
```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
...
+
+4. (optional) The 'patternfly.charts' module is not a dependency in the default angular 'patternfly' module.
+   In order to use patternfly charts you must add 'patternfly.charts' as a dependency in your application:
+
+        my-app.module.js:
+
+        angular.module('myApp', [
+          'patternfly',
+          'patternfly.charts'
+        ]);
```
Comment (jeff-phillips-18) [inline README.md]: Only the patternfly.charts module is optional. I'm wondering if we should show the addition of 'patternfly' to the myApp module. Pretty basic and I'd think angular devs would know to do this but calling it out for charts leads me to think we should call it out for the base.

Comment (dtaylor113) [inline README.md]: I'm not sure what you are suggesting?

Comment (jeff-phillips-18) [inline README.md]: I'm suggesting we might want to add, adding the patternfly module to the dependencies:

angular.module('myApp', [
         'patternfly'
       ]);

and then state that optionally, add the charts dependency.

**Comment by jeff-phillips-18** [comment]
LGTM. Issue above was more of a question than anything else.

File: README.md [hunk 2]
```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
...
 ```
 
+### Using Angular-PatternFly In Your Application
+
+Note:
+
+1. Add Angular-PatternFly as a dependency for your project and you'll receive all the libraries you'll need:
+
+        $ bower install angular-patternfly --save
+
```
Comment (erundle) [inline README.md]: I don't think this gives you everything since they still need to add in angular etc.

File: README.md [hunk 3]
```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
...
+2. Add the follwing CSS includes to your HTML file(s), adjusting path where needed:
```
Comment (erundle) [inline README.md]: Include not includes

File: README.md [hunk 4]
```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
...
+
+        <!-- Angular-PatternFly Styles -->
+        <link rel="stylesheet" href="bower_components/angular-patternfly/dist/styles/angular-patternfly.min.css" />
+
+3. Add the following script includes to your HTML file(s), adjusting where necessary to pull in only what you need:
+
+        <!-- Angular-PatternFly  -->
+        <script src="bower_components/angular-patternfly/dist/angular-patternfly.min.js"></script>
+
+        <!-- C3, D3 - Charting Libraries -->
```
Comment (erundle) [inline README.md]: I would put something here saying only required if you are using the patternfly.charts module

**Comment by dtaylor113** [comment]
Hi, addressed Eleni's comments.

Still have the question if we require angular-bootstrap?
<script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.min.js"></script>

Thanks,
- Dave

**Pull request merged**